<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="start_window" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>289</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <widget class="QLabel" name="start_img">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="boton1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="game_scene" native="true">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>310</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>31</y>
       <width>231</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
